babel.config.js

```diff
@@ -7,8 +7,7 @@ module.exports = api => {
         root: ["./"],
         alias: require("./aliases.json"),
       },
-    ],
-    ["react-native-reanimated/plugin"],
+    ]
```

Comment:
@mohitharshan123 after removing this please make sure that neetoUIRN works if run directly as well. Merge if checked already.

Reply:
@sangameshsomawar neetoUIRN was not working even without removing the plugin. I guess I can take it up in seprate issue when updating RN version.

Reply:
@mohitharshan123 ok got it. Finally we got a fix !!